packages/gatsby/cache-dir/gatsby-browser-entry.js

```diff
@@ -71,7 +71,7 @@ useStaticQuery(graphql\`${query}\`);
 `)
   }
 
-  if (context[query] && context[query].data) {
+  if (context && context[query] && context[query].data) {
```

Comment:
We could since this code is imported along with user land code and will get transpiled when it is included in the site's bundle. I left it as is in the interest of reducing churn though. Let's refactor later?